### PR TITLE
feat/issue-#288 fix

### PR DESCRIPTION
**Pull request by ekumamatthew**
feat/issu-#288 fix for test_zklend_initializer

### CODE REVIEW

File: apps/data_handler/tests/conftest.py [hunk 1]
```diff
@@ -32,6 +32,8 @@ def mock_initializer_db_connector() -> None:
     :return: None
     """
     mock_initializer_connector = MagicMock(spec=InitializerDBConnector)
+      mock_initializer_connector.engine = MagicMock()
```
Comment (djeck1432) [inline apps/data_handler/tests/conftest.py:35]: Please, fix spacing

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@ekumamatthew Please, sync with `master` branch

**Comment by djeck1432** [comment]
Screenshot 2024-11-23 at 20 53 32
Do you need help to sync with master?

**Comment by djeck1432** [comment]
@ekumamatthew

**Comment by ekumamatthew** [comment]
It's in sync with master.

**Comment by ekumamatthew** [comment]
I'm closing this issue because i opened a new PR  https://github.com/CarmineOptions/derisk-research/pull/306 .

**Pull request closed without merging**